smartsim/_core/launcher/colocated.py

```diff
@@ -111,26 +114,43 @@ def _build_colocated_wrapper_cmd(
     # the lock on the file.
     lockfile = create_lockfile_name()
 
+
+
+
     # create the command that will be used to launch the
     # database with the python entrypoint for starting
     # up the backgrounded db process
-    cmd = [
-        sys.executable,
-        "-m",
-        "smartsim._core.entrypoints.colocated",
-        "+lockfile",
-        lockfile,
-        "+db_cpus",
-        str(cpus),
-    ]
+
+    cmd = []
+    cmd.extend(
```

Comment:
is there a reason for the change from declaring a populated list to declaring an empty list and then extending?

Reply:
Good point and the answer is no.